### DISCUSSION TITLE
RPi-400 will not boot with CaribouLite connected

### DISCUSSION THREAD

**Discussion by kcman100** [cariboulabs/cariboulite#99] [Q&A]
Trying to use the CaribouLite with a RPi400.
Seems to build okay.

I get:
[ERROR] cariboulite_init_driver() failed
But I assume that is just because I do not have the CaribouLite connected yet?

With the CaribouLite connected, the RPi-400 goes blank after a fraction of a second flash of the RPi Splash screen.

Any ideas how to get the CaribouLite working?
I have tried the develop_R1 and konimaru branches/forks, no difference.

Thanks,

Ken

**Comment by konimaru** [comment]
Sounds like issue #88. Can you post your **dmesg** output (you may have to login over network)? Also the output from
```
cat /proc/iomem
```
for both cases (plugged in and not) would be interesting.

**Comment by kcman100** [comment]
[rpi400_no_display.tar.gz](https://github.com/cariboulabs/cariboulite/files/11073075/rpi400_no_display.tar.gz)

Logs attached.
I was not thinking the RPi was alive at all, so I did not try network access.  Interesting that display was dead, but OS+networking was running.

Thanks for taking a look!  -Ken

**Comment by kcman100** [comment]
I did a clean OS for my RPi-400.  Latest desktop 64-bit load from RPi-Website.
And now seems to be working.  cariboulite_test_app runs.
This was an upgrade from 10-Buster to 11-Bullseye (both were 64-bit).
And I suppose a chance something I had previously loaded on 10-Buster was causing a problem?

**Reply by konimaru** [reply]
Thanks for the logs. I suspected as much, dug out my 400 and (with latest OS image) couldn't find any reference to **firmwarekms**. The **fe600000** slot was empty as well. So yes, there is/was a clash with **smi**. Glad you got it working.
```
23c23
< 00000000-00000000 : fe600000.smi smi@7e600000
---
> 00000000-00000000 : fe600000.firmwarekms firmwarekms@7e600000
```
Something to keep an eye on ...
